File: programmers/java/lev0/한 번만 등장한 문자.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Set;
class Solution {
    public String solution(String s) {
        List<String> list = new ArrayList<>();
        for(var tmp : s.split("")){
            if(s.lastIndexOf(tmp)==s.indexOf(tmp)){
                list.add(tmp);
            }
        }

        Collections.sort(list);
        return  String.join("",list);
    }
}

File: programmers/js/lev0/한 번만 등장한 문자.js
function solution(s) {
  let dict = {};
  s.split("").forEach((el) => {
    dict[el] ? (dict[el] += 1) : (dict[el] = 1);
  });

  return Object.keys(dict)
    .filter((el) => dict[el] == 1)
    .sort()
    .join("");
}

function solution(s) {
  let res = [];
  for (let c of s) if (s.indexOf(c) === s.lastIndexOf(c)) res.push(c);
  return res.sort().join("");
}
